docs/integrations/destinations/pinecone.md

```diff
@@ -74,6 +74,7 @@ OpenAI and Fake embeddings produce vectors with 1536 dimensions, and the Cohere
 
 | Version | Date       | Pull Request                                                  | Subject                                                                                                                                              |
 |:--------| :--------- |:--------------------------------------------------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------|
+| 0.0.25 | 2023-05-01 | [#37601](https://github.com/airbytehq/airbyte/pull/37601) | Add support for Pinecone Serverless|
```

Comment:
Ditto above. Suggestion to bump version to `0.1.0`.

Also, suggesting we remap the PR number to this newer one and bump release date to today. 🤞 

```suggestion
| 0.1.0 | 2023-05-06 | [#37756](https://github.com/airbytehq/airbyte/pull/37756) | Add support for Pinecone Serverless |
```